content/docs/lessons/foundations/lesson-one.md

```diff
@@ -54,14 +54,29 @@ a lot of powerful things later on. In this case, + _is the
 The **operator** decides how the rest of the **elements** are **evaluated**.
 
 {{< scheme >}}
+(+ 1 2)
+(+ 1 2 3 4)
+{{< /scheme >}}
+Now with Multiplication and Division...
+{{< scheme >}}
+(* 2 4)
+(* 2 4 2)
+
 (/ 20 4)
 (/ 20 4 2)
+(/ 5 2)
 {{< /scheme >}}
 
-Play with the above example. You'll notice that the order really
-matters here - the first element, "20", is successively divided by
+Play with the above example. You'll notice that depending on the **operator**,
+order really matters here - the first element, "20", is successively divided by
 every other element in the list.
 
+You may also notice that expressions such as (/ 5 2) will evaluate to 5/2. Try
+the below instead {{< scheme >}} (/ 5 2.0) {{< /scheme >}}
```

Comment:
Nice, was thinking of including this, but thought it may bring up types too early.